_posts/2016-07-16-leaflet-1.0-rc2.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: Announcing Leaflet 1.0-rc2
+description: Leaflet 1.0 Release Candidate 2 is out
+author: Yohan Boniface
+authorsite: http://yohanboniface.me
+---
+
+Here comes the second release candidate for Leaflet 1.0,  the more stable Leaflet version ever!
```

Comment:
If the question is "did I intend the sentence to be odd", the answer is no :p
But you know, my English… ;)
How would you express that, then? `stablest` seems a bit odd to me too.